src/components/events-card-new/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { tailwindWrapper } from "../../../../../formula_one/src/utils/tailwindWrapper";
```

Comment:
```suggestion
import React, { Component } from 'react';

import { tailwindWrapper } from "../../../../../formula_one/src/utils/tailwindWrapper";
```

We should always leave a line between 'react' imports and 'formula-one' imports

Reply:
done changes